api/public/public_api/endpoints/google_stt.py

```diff
@@ -47,13 +59,14 @@ def post(self):
         stt_response = self._call_google_stt()
         response = self._build_response(stt_response)
         self._write_stt_result_file(response)
+        self._update_account_activity_ts()
+        self._track_account_activity()
```

Comment:
Will this update for empty data, which would be a sign of false activation?

Reply:
Added a check for an empty response from the STT API call.

Reply:
Um where is this check?